automacent-fwk-components/automacent-fwk-core/src/main/java/com/automacent/fwk/core/Driver.java

```diff
@@ -345,9 +356,12 @@ public void startDriver(DriverManagerType driverManagerType) {
 				BaseTest.getTestObject().getTimeoutInSeconds()));
 		webDriver.manage().timeouts().setScriptTimeout(getScriptTimeoutInSeconds(), TimeUnit.SECONDS);
 		_logger.info(String.format("Script timeout set on driver to %s seconds", getScriptTimeoutInSeconds()));
-		webDriver.manage().window().maximize();
-		webDriver.manage().deleteAllCookies();
-		_logger.info("Cookies deleted");
+
+		if (BaseTest.getTestObject().getDebuggerAddress().isEmpty()) {

```

Comment:
Add condition `|| !driverManagerType.name().equals(DriverManagerType.CHROME.name())`. If by any change we gave browser as FIREFOX and set debuggerAddress="", this code block will not get executed without the suggested condition.

Reply:
added